cmd/internal/dumplogs/dumplogs.go

```
@@ -214,25 +212,12 @@ func (c *dumpLogsCommand) dumpLogsForEnv(ctx *cmd.Context, statePool *state.Stat
 		if !ok {
 			break
 		}
-		_, _ = writer.WriteString(c.format(
-			rec.Time,
-			rec.Level,
-			rec.Entity,
-			rec.Module,
-			rec.Message,
-			rec.Labels,
-		) + "\n")
+		line, err := json.Marshal(rec)
+		if err != nil {
+			return errors.Trace(err)
+		}
+		_, _ = writer.WriteString(fmt.Sprintf("%s\n", line))
```

not sure why we ignore the error here, but it matches the previous code.